sources/main.swift

```diff
@@ -2,6 +2,19 @@ import CoreLocation
 import Foundation
 
 var arguments = CommandLine.arguments.dropFirst()
+
+var deviceName: String?
+if let argumentIndex = arguments.index(of: "-s") {
+    let device = arguments.suffix(from: argumentIndex).dropFirst().joined(separator: " ")
+    if device.isEmpty {
+        exitWithUsage()
+    } else {
+        deviceName = device
+    }
+
+    arguments = arguments.prefix(upTo: argumentIndex)
```

Comment:
Will this still support arguments out of order? 
Like: 
`-s iPhone 8 -q Lyft HQ`

Reply:
No, but that was kinda intentional, I don't want our arg parsing to get too crazy, and this way people can continue to not worry about quoting anything which I think is nice. I think if we wanted to allow these to be in either order, I would say we should force all arguments to be quoted